lib/r10k/environment/git.rb

```diff
@@ -50,7 +50,7 @@ def initialize(name, basedir, dirname, options = {})
       :ref     => :self,
       :remote  => :self,
 
-    }, raise_on_unhandled: false)
+    }, raise_on_unhandled: true)
```

Comment:
Is this related to the comment below? If so, are you thinking we ought to cut a branch (i.e. do a major) for this PR to go into? Or do you think the caution there was overblown?

Reply:
This still being here is an oversight. I was experimenting with changing the args passed here and wanted to use this to validate that wip.